Rubberduck.VBEEditor/QualifiedModuleName.cs

```diff
@@ -69,6 +93,7 @@ public QualifiedModuleName(string projectName, string projectPath, string compon
             ProjectId = $"{_projectName};{ProjectPath}".GetHashCode().ToString(CultureInfo.InvariantCulture);
             _componentName = componentName;
             ComponentType = ComponentType.ComComponent;
+            ModuleContentHashOnCreation = GetModuleContentHash(null);
```

Comment:
is it impossible to obtain the component here? Because if the GC behaviour and the reference-retention is different depending on which constructor we use to create QMNs, that's a big no-no.

An alternative might be to deprecate this ctor, but I have no clue how often it's used :worried:

Reply:
This constructor is used for the QMNs for the components of referenced COM libraries.